packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -799,9 +835,150 @@ export const RemoteServerFactory = {
+        }
+
+        const _serverlessExpressHandler = serverlessExpress({
+            app,
+            // the underlying serverless-express library strips `headers`
+            // which our ssr-infrastructure expects, these mappings restore
+            // and pass through all request / response properties
+            eventSource: {
+                getRequest: ({
+                    event,
+                    method = event.httpMethod,
+                    path = getPathWithQueryStringParams({event})
+                }) => {
+                    let headers = {}
+
+                    if (event.multiValueHeaders) {
+                        headers = getCommaDelimitedHeaders({
+                            headersMap: event.multiValueHeaders,
+                            lowerCaseKey: true
+                        })
+                    } else if (event.headers) {
+                        headers = event.headers
+                    }
+
+                    let body
+
+                    if (event.body) {
+                        body = getEventBody({event})
+                        const {isBase64Encoded} = event
+                        headers['content-length'] = Buffer.byteLength(
+                            body,
+                            isBase64Encoded ? 'base64' : 'utf8'
+                        )
+                    }
+
+                    const remoteAddress =
+                        (event &&
+                            event.requestContext &&
+                            event.requestContext.identity &&
+                            event.requestContext.identity.sourceIp) ||
+                        ''
+
+                    return {
+                        method,
+                        headers,
+                        body,
+                        remoteAddress,
+                        path
+                    }
+                },
+                getResponse: ({statusCode, body, headers, isBase64Encoded}) => {
+                    const multiValueHeaders = {}
+
+                    Object.entries(headers).forEach(([headerKey, headerValue]) => {
+                        const headerArray = Array.isArray(headerValue)
+                            ? headerValue.map(String)
+                            : [String(headerValue)]
+
+                        multiValueHeaders[headerKey.toLowerCase()] = headerArray
+                    })
+
+                    return {statusCode, body, headers, multiValueHeaders, isBase64Encoded}
+                }
+            },
+            // handler,
+            // binarySettings: {
+            //     isBinary: ({headers}) => true,
+            //     contentTypes: binaryMimeTypes,
+            //     contentEncodings: []
+            // },
+
+            // TODO: this is for debugging, remove
+            respondWithErrors: true,
+            resolutionMode: 'CALLBACK',
+            callback: (err, response) => {
```

Comment:
@olibrook @bfeister I don't understand how this `callback` function would get propagated to the `proxy` function in the library code https://github.com/vendia/serverless-express/blob/mainline/src/configure.js#L36

Do you understand or do you think it is deprecated in the library?

Reply:
Oh the callback needs to be passed on every request https://github.com/vendia/serverless-express/blob/mainline/src/configure.js#L99-L103. 

I don't see how it can be passed when instantiating the  serverless-express object anymore (like we previously did)...

Reply:
Instead of passing the callback in here I think we have to bind the callback to the handler method we return from the `_create_handler` method.

Right now I don't think the callback function we have defined is being used.

Reply:
This is a great catch @lizcross investigating now!